src/ggrc-client/js/plugins/utils/models-utils.js

```diff
@@ -158,6 +159,32 @@ function getCustomAttributableModels() { // eslint-disable-line
     .map((modelName) => businessModels[modelName]);
 }
 
+
+async function initAuditTitle(instance, isNewInstnace) {
+  const isAuditTitle = (
```

Comment:
Previously, method, which contained this logic was called `is_audit_modal`. This variable saves the logic, which was in those method, so I think it should be called `isAuditModal`, not `isAuditTitle`. And inside function `initAuditTitle` I suppose to see variable `isAuditModal` rather than `isAuditTitle`
```suggestion
  const isAuditModal = (
```

Reply:
@DimaDK24 Actually, I don't see the reason to name this var like `isAuditModal`. Right now, this is a standalone function without connection with modals - this method as a module shouldn't know how'll call it cos of it's like some lib util - it may be potentially used in several placed - not only in models (YAGNI is not violated). In order to prevent high coupling with modals, better to leave the name as it is for now. We can name `isAudit` as well - in this case it doesn't matter IMO.

Reply:
@iSokrat, yeah, agree with you regarding modals. But I still don't like `title` part here. Probably the best name would be `isAudit` or `isAuditInstance`, because here we directly check that `instance` is instance of Audit model

Reply:
@DimaDK24 Will take a look.

Reply:
Will use `isAuditInstance`.